game/scenes/Christine.gd

```diff
@@ -75,6 +75,8 @@ func _physics_process(_delta):
 		if not $DigSFXPlayer.playing and chop_area.get_overlapping_bodies().size() > 0:
 			$DigSFXPlayer.play()
 		current_state = State.CHOP
+	if Input.is_action_just_released("chop"):
+		$DigSFXPlayer.stop()
 
```

Comment:
Maybe you can add the flickering of the beech outlining and the chop sound here. This would readily close some bug tickets.

Reply:
I just realised the sound bug is already solved in this PR. But maybe can also add the flickering. Speaking of flickering, is it still need now that we have the progress bar? Maybe it's easier to keep only the selection outlining.

Reply:
chop sound is already in there? or what do you mean?
I reset the beech state with the new commit, so the flickering should also stop